src/Reports/SiteSummary.php

```diff
@@ -55,16 +59,20 @@ public function columns()
     public function getReportField()
     {
         Requirements::css('silverstripe-maintenance/css/sitesummary.css');
-        $grid = parent::getReportField();
-        $config = $grid->getConfig();
+
+        /** @var GridField $gridField */
+        $gridField = parent::getReportField();
+
+        /** @var GridFieldExportButton $exportButton */
+        $exportButton = $gridField->getConfig()->getComponentByType(GridFieldExportButton::class);
```

Comment:
Although `getConfig` is relatively trivial, I think the way it was stored to a variable before was better in repeated use.
Could you please shift the `getComponentByType` call to use `$config` (after it's defined)?